lib/block_view_dao_coin_limit_order_test.go

```diff
@@ -1411,7 +1411,7 @@ func TestDAOCoinLimitOrder(t *testing.T) {
 		m0DESOBalanceNanosAfter = _getBalance(t, chain, mempool, m0Pub)
 		m1DESOBalanceNanosAfter := _getBalance(t, chain, mempool, m1Pub)
 		require.Equal(m0DESOBalanceNanosBefore, m0DESOBalanceNanosAfter)
-		require.Equal(m1DESOBalanceNanosBefore-uint64(34), m1DESOBalanceNanosAfter) // Hard-coded fee.
+		require.Equal(m1DESOBalanceNanosBefore-uint64(35), m1DESOBalanceNanosAfter) // Hard-coded fee.
```

Comment:
I *think* we can get the fee from DAO coin limit order txns: 
```
testMeta.txns[len(testMeta.txns)-1].TxnMeta.(*DAOCoinLimitOrderMetadata).FeeNanos
```

maybe we write a little func to get this so we can get rid of hard coded fees?

Reply:
alternatively, we update all the helpers like `_doDAOCoinLimitOrderTxn` to return fees and then append them to a list of fees on TestMeta?

Reply:
I went w/ your first suggestion. See most recent commit.